http/src/request/channel/invite/create_invite.rs

```diff
@@ -47,6 +48,15 @@ pub struct CreateInvite<'a> {
     reason: Option<String>,
 }
 
+#[derive(Debug)]
+/// The error returned when an invite cannot be created.
+pub enum CreateInviteError {
+    /// The error is returned if `max_age` is greater than 604800.
+    MaxAgeInvalid(String),
+    /// The error is returned if `max_use` is greater than 100.
+    MaxUseInvalid(String),
+}
+
```

Comment:
When targeting 0.4, we need to use the new error impl pattern. I'll be making a template of this soon, but before then, please refer to #691 on how to structure twilight errors.